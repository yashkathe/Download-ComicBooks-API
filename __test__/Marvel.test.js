const { getMarvelComics } = require('../index');


test("Marvel Comics", async () => {
    expect(await getMarvelComics(1)).toMatchSnapshot();
});

