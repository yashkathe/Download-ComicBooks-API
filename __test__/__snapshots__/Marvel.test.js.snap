// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Marvel Comics 1`] = `
[
  {
    "coverPage": "https://i0.wp.com/getcomics.info/share/uploads/2022/10/Ghost-Rider-Epic-Collection-Vol.-1-Hell-on-Wheels-2022.jpg?fit=400%2C615&ssl=1",
    "description": "In 1972, one of the most iconic characters in comic book history, the flame-skulled Ghost Rider, burned his demonic presence into readers’ minds. Forevermore, a legion of fans were addicted to the Rider’s combination of hell-on-wheels drama and action-horror adventure. And it begins here when Johnny Blaze makes a deal with the devil to save his friend’s life. The payment due? Transformation into the Ghost Rider. The stories that follow will take the horror hero to Hell to battle Satan, pit him against the tempting Witch-Woman and team him with Daimon Hellstrom, Son of Satan. Written by Gary Friedrich and Tony Isabella and gloriously illustrated by Mike Ploog, Tom Sutton and Jim Mooney. COLLECTING: Marvel Spotlight (1971) 5-12, Ghost Rider (1973) 1-11, Marvel Team-Up (1972) 15",
    "downloadLinks": [
      {
        "Download Now": "https://getcomics.info/links.php/5POUcYVJ3b4tr45V7uFcW/TP30zjq1hVdr8cIL2SX2mjvJOQe96zzxJiIeUl726BN51sJwkZ/6yg/1mkHUFViM22TgOdShD0PXzdX47OjzbcPgnLvUEywWafZrVt8ptvBg+8FgFrq/0zE1i8b1LokStQBG8XQxdFfZTzcw02gt4FasW2CALN569OZGdYSFmSeehvzvm/mSd779GBC9pDIA==:mcywEZ+GdqdW3OhWS0JR7g==",
      },
      {
        "UFILE": "https://getcomics.ufile.io/ila57kot",
      },
      {
        "MEGA": "https://getcomics.info/links.php/LA8EKdaWD8VWzOHPf8QSamcQusz9j2An7s7CmSvhaH27QcXcJZAvBRy6t+idxKsOjoEp45AZOObnwQb8XpcpFVCdD6dqs5y4Ui2QISIDNh4=:CYt5mzJYbSk7fSigFc5ZVA==",
      },
      {
        "MEDIAFIRE": "https://getcomics.info/links.php/yngP0anf3d8XnkS5zWLBApESL/rBfG0gc5NBIZ6ODp+n+bGO1hJcqHehVwYXx0IL2JsQxgaGErYivUCwzleG57jK2V44D3ymN2DxKNJ6PPzpd6D3HBTZvEx401NFwpqG8sASgixo6dBZPrp5HcyEe18nyvgAIpBpbZzuuVXB70QrBfZl7813PQ8SXoczS/wM:xICmspz5BLzmOnjRx4RDfQ==",
      },
    ],
    "info": {
      "ImageFormat": "JPG",
      "Size": "697 MB",
      "Year": "2022",
    },
    "title": "Ghost Rider Epic Collection Vol. 1 – Hell on Wheels (2022)",
  },
  {
    "coverPage": "https://i0.wp.com/getcomics.info/share/uploads/2021/06/Ultimate-Comics-Doomsday-TPB-2011.jpg?fit=400%2C615&ssl=1",
    "description": "Who is the face of true evil in the Ultimate Universe? What is the biggest secret the world doesn’t know? Join the gathered heroes as they embark on an adventure unlike anything seen in Marvel Comics — and that’s a promise! Collects Ultimate Comics Enemy #1-4, Ultimate Comics Mystery #1-4, Ultimate Comics Doom #1-4.",
    "downloadLinks": [
      {
        "Download Now": "https://getcomics.info/links.php/UX7srJOrAEo8Ppg97TK0ovvsB44leVslSkEGaDQOzO0zObvAYLp86yBVQTq29Ty2elTr/wcxaz+98vYhmNpVELFTee4QtAFzSRtiTSip4sPBJuBSn/MtboTctdPUd8aiqASthMVtDDDnt5Vgwz4B9Nq0rUdPpxfF0ujD/+k23BNVNXtqIrVZCmlZHBbS3tSa:hulJt6hlozP8IaMkTq9iIw==",
      },
      {
        "UFILE": "https://getcomics.ufile.io/k4d3ci4w",
      },
      {
        "MEGA": "https://getcomics.info/links.php/4uu2LldDcL0yfcI4e26VDClp2SY9uzsVQEnN1y34TjmzjwZYTFSRINRwuubvu1gGVKFGj4TmyHhYqtu4T5Pe481iADllGDPM8QEHoYpQxC4=:Dc1CKMgvTFhgYTrJvOwO2A==",
      },
      {
        "MEDIAFIRE": "https://getcomics.info/links.php/S2qTIFTxaPbipmJ7N8b8M+ZSlOdk0AAZT7XJ5l30zO2D+kZ56hZM9yMjWYieKVjVKia1S073npJGVTzg8P+CUQ/ucyeo5BqD/gVYXh8S48WD3IsmHjKi3XkkqunmCGAiWv/z8M2R6cLT+kJQUgh0KLTmHwOJG+yQfhK0WjJQV/Fgq0VV85qpD6W3Msaxea3T:bQirhdzB0g4xUVHulMD73A==",
      },
      {
        "ZIPPYSHARE": "https://www50.zippyshare.com/v/QNwEqagW/file.html",
      },
    ],
    "info": {
      "ImageFormat": "JPG",
      "Size": "473 MB",
      "Year": "2011",
    },
    "title": "Ultimate Comics Doomsday (TPB) (2011)",
  },
  {
    "coverPage": "https://i0.wp.com/getcomics.info/share/uploads/2022/10/Shang-Chi-By-Gene-Luen-Yang-Vol.-3-Family-Of-Origin-TPB-2022.jpg?fit=400%2C615&ssl=1",
    "description": "Prepare to learn the secrets of Shang-Chi’s mother! Arrows will definitely fly when we flash back to the first meeting of Shang-Chi’s parents — and we aren’t talking about the arrows of Cupid! Who exactly was the woman who gave birth to the legend? And how could someone so virtuous and heroic possibly fall for an evil warlord like Zheng Zu? All will be revealed — and the answers will surprise you! Plus, in the present, the tides of fortune begin to turn against Shang-Chi, as the mastermind that has been gathering all of his foes stands revealed at last! Collects Shang-Chi (2021) #7-12, Marvel’s Voices: Identity (2021) #1 (Shang-Chi story).",
    "downloadLinks": [
      {
        "Download Now": "https://getcomics.info/links.php/62dobwN6DQMej5ftk/qFyiHmk9YPgtlomI/wgutRqP6wRdjaVan7vhZ34zMPmc8DAH6nyp/HKRGCs4v4RTDDnpesXOHJPtS2/sUTfIC9z0XIV2ILM+yi8S4mLBl0s+1uBha7/uSZGUKrsZbPR82V2+OYAvz6OWbHklrC5iAVNGuYLIDgUfRBZkOXNft1GxOkL1ooPJM2OgTOhqrc3aP9UQ==:MlzDNmjcHKdR9WHgwRHf3Q==",
      },
      {
        "UFILE": "https://getcomics.ufile.io/n3v6pesw",
      },
      {
        "MEGA": "https://getcomics.info/links.php/OzyIfqCkZIHopo6Efy2CQwxk+niHq+o4GgNw5q+l++t4AIpzJrn0uIchY7AijGMXM2d4pUdZ57eufXKAALXW3uBxSoB/HHEdaI61CExKugM=:ARFvrSyh7o2QlqDcilrikQ==",
      },
      {
        "MEDIAFIRE": "https://getcomics.info/links.php/2xiP2Txp8Dpv2XQ7DnS3uWQXECVe71KBGmsGVSt8GEq7Au+uFJuSOtzsVsDJ6JMaig8DOTGWTCHU4Ty3YW/nS44vePyAddMHScK6YBVlg0tL8MAEMB7tXSKRsOlrPDCKUSYGbbFhZ82OOr9Au32V68YIm+VK2GzNoBqwk+UZyiryhdlqJ8FZevq9w86t9nPZ:HPzlJDhREoR/RqdD5mDKmQ==",
      },
      {
        "ZIPPYSHARE": "https://www35.zippyshare.com/v/8wTS5OTt/file.html",
      },
      {
        "Read Online": "https://readcomicsonline.ru/comic/shangchi-2021",
      },
    ],
    "info": {
      "ImageFormat": "JPG",
      "Size": "233 MB",
      "Year": "2022",
    },
    "title": "Shang-Chi By Gene Luen Yang Vol. 3 – Family Of Origin (TPB) (2022)",
  },
  {
    "coverPage": "https://i0.wp.com/getcomics.info/share/uploads/2022/10/Miracleman-Omnibus-2022.jpg?fit=400%2C600&ssl=1",
    "description": "Middle-aged reporter Michael Moran always knew he was meant for something more. When an unexpected series of events leads him to reclaim his destiny, Miracleman is reborn! But Miracleman’s return threatens to unravel Moran’s life. Their connections to Dr. Emil Gargunza and Project Zarathustra bring with them disturbing revelations and trigger the return of a childhood friend who, beneath his sinister smile, has become something terrifying. Pushing the concept of the super hero to its logical conclusion, MIRACLEMAN is nothing short of a revelation. The series reinvented the super hero and 40 years later stands as one of the influential works in the comic book artform. This first-ever Omnibus edition collects MIRACLEMAN BOOKS ONE through THREE, complete with a massive trove of covers, original artwork and rare features. COLLECTING: Warrior (1982) 1-18, 20-21; Miracleman (1985) 1, 3, 6-16; Marvelman Special (1984) 1; A1 (1989) 1 [as presented in Miracleman (2014) 1-16]; All-New Miracleman Annual (2014) 1",
    "downloadLinks": [
      {
        "UFILE": "https://getcomics.ufile.io/kaew3o4m",
      },
      {
        "MEGA": "https://getcomics.info/links.php/GaDN11A4Z3YIaUgcy5W/T/xYYeP0ZrFFn0Xbuvnjft50aSbDGwXw/j1shYPZ+EgjHivKuXYYEm1Vgd9UKxGns6xpG03cYzRLbdFsKwA1J+A=:U0BrZJCInH61gyR8vHFsrQ==",
      },
      {
        "MEDIAFIRE": "https://getcomics.info/links.php/XsKq+CtokuyRFTbxOMKVkeYMxuYMAxW/HQUtkanR3r80pxtkghr80h+7Hrws+/VovWAvH4k+NRWLJkOEjFZn8mSWX0LRjR9dY5gHKOiGU/KlGiCCogLWLB5/E9B2KShwr6LcmMb7y149cgVRB9Q5pd8wllbs90e7iazjs3Knds5wrk4uPiSMkOzLjFsU8+hc:kZrPlg31ETvORRelcTvslw==",
      },
      {
        "Read Online": "https://readcomicsonline.ru/comic/miracleman-omnibus-2022/HC",
      },
    ],
    "info": {
      "ImageFormat": "JPG",
      "Size": "1.3 GB",
      "Year": "2022",
    },
    "title": "Miracleman Omnibus (2022)",
  },
  {
    "coverPage": "https://i0.wp.com/getcomics.info/share/uploads/2022/10/Avengers-by-Jason-Aaron-Vol.-10-The-Death-Hunters-TPB-2022.jpg?fit=400%2C615&ssl=1",
    "description": "Deep in the dark heart of the multiverse stands a shining watchtower. From there, the mysterious Avenger Prime deploys his operatives — the cybernetic soldiers called the Deathloks — to deal with grave threats to all the myriads realms that make up reality. But a hideous force has risen to oppose him, and these Multiversal Masters of Evil are the most vicious and dangerous foes any universe has ever seen. Now, the Deathloks have come to the Marvel Universe bearing a dire warning for the Avengers — and the Death Hunt is about to begin! For savage hunters follow close behind to ensure that the Deathloks’ warning of megaversal doom is never delivered. And their first stop is the golden realm of Asgard… COLLECTING: Avengers (2018) 51-55, Free Comic Book Day 2021: Avengers/Hulk 1 (Avengers story)",
    "downloadLinks": [
      {
        "Download Now": "https://getcomics.info/links.php/t4TJyW3G5Li2y/2ES1j/Jjx4aPkB+6lQM5IHmahKukbiLpIJTcEDdo+/nbZYvirRcrWsnSiiwqIwiNPrE5sEPdRRKDYwkj0wt+f0ryy5LQfvPJRLZjETmIF2qgDOfY7i7jbOGsVGpuYBtpkZrc8rhO/G9TJnUs7Sk2mpC5b0i15gPQDgOZuavgAb1unawRo92T6mNUZMPEFkSFIPbLeitA==:eSEN5egchSlmMcB0vvcgGA==",
      },
      {
        "UFILE": "https://getcomics.ufile.io/a30rkybo",
      },
      {
        "MEGA": "https://getcomics.info/links.php/sIjD+o3DsQmwM7W/k7jKU7U5ZumGt4NyNDQkk9oaY6PbX6shUT6lt8NejH5eFEn20HJAsfdRhzA8m7EmC6iUtX5TiYd9G4bCFjcT9C58/Rg=:IgyVI84JEz1510DpewuTWA==",
      },
      {
        "MEDIAFIRE": "https://getcomics.info/links.php/LrI1EV2AZ/wCq/0w1akert1aDGBOm624cP1aJjDWMWb6QbxQB3kOpE1048IGRkEjCPGGLNmKrUbEu1QiImMj+st6FfD5mQoe3Bm1ltiebTOtmPmAhmoH6Uz07FqnvvnMLsKXUnarYDtkH58bkTNwaLtfi1XqWvek/TQB8MEeKvVnoz6X2KhS4bgbxtJcEePkxLZcEmODsWVAS7VUgPPhaQ==:ECqsyxDuH6lyamuMpBArcg==",
      },
      {
        "ZIPPYSHARE": "https://www3.zippyshare.com/v/Q61JHwH8/file.html",
      },
      {
        "Read Online": "https://readcomicsonline.ru/comic/avengers-2018",
      },
    ],
    "info": {
      "ImageFormat": "JPG",
      "Size": "286 MB",
      "Year": "2022",
    },
    "title": "Avengers by Jason Aaron Vol. 10 – The Death Hunters (TPB) (2022)",
  },
  {
    "coverPage": "https://i0.wp.com/getcomics.info/share/uploads/2022/10/X-Terminators-2-2022.jpg?fit=400%2C608&ssl=1",
    "description": "REFLECTIONS OF DEADLY VENGEANCE! Four deadly X-Women find themselves held captive and fighting for their lives…and more importantly, fighting to get revenge on the @#$%# dead man who did this to them!",
    "downloadLinks": [
      {
        "Download Now": "https://getcomics.info/links.php/z2SG0jzqwnq2T5yoacKxetAKpiyFRFl8ANhDZ4SSrgXNcKKK2WeLpy90yVEApUYDRjm1L7pPuw4kQXZzf1CUEeZ/M64C7bOlb50bnNkOglqmzoKTOlybp7zsQRA/DscFwoUYKRJTdwB/mmWeCZwyFst7OqbFiFTnV0woCGBGK7o=:tjoVFhZHaq/IGojct+86Ng==",
      },
      {
        "UFILE": "https://getcomics.ufile.io/p48m7btc",
      },
      {
        "MEGA": "https://getcomics.info/links.php/UJNa2yM8UDZKdzvXU5Xk0sebtqNQq+5S+NDctYiMSj0SvAfgGkmQG5sXXD93vhr6dMXO9n4gt2fs8Hr+4v/AddSUSxlEb9JPJKaLBWYvlxQ=:2h1KDNR4oK2RTTe45x+MHw==",
      },
      {
        "MEDIAFIRE": "https://getcomics.info/links.php/9ypwyhxxuB1kFMY9kzIEuXQGiLbaXDZWiTfcULolbgeTAsB/Xr1WoAZ8SVkNu3kZK1vz85e+pY9zpGe4zvFxH7zlwZsV/fhpESJ8ZSuEkaIjMk/C+c/SSFb+jInmYZIU0yjfexN9SaPENSAX9tp2Bk4+dR9tZuBORlD0/Xf73tPeuExqw6Zm2DLSdKVbok1C:oB3n6zH3YPLbG5gpblT+uw==",
      },
      {
        "ZIPPYSHARE": "https://www3.zippyshare.com/v/zEcvFN7U/file.html",
      },
      {
        "Read Online": "https://readcomicsonline.ru/comic/xterminators-2022/2",
      },
    ],
    "info": {
      "ImageFormat": "JPG",
      "Size": "35 MB",
      "Year": "2022",
    },
    "title": "X-Terminators #2 (2022)",
  },
  {
    "coverPage": "https://i0.wp.com/getcomics.info/share/uploads/2022/10/X-Men-Legends-3-2022.jpg?fit=400%2C608&ssl=1",
    "description": "CAN THE X-MEN SURVIVE MOJOWORLD? IT’S A LONGSHOT! Ann Nocenti returns to her co-creation with an all-new story set after the original LONGSHOT miniseries! Longshot’s adventure on Earth may be over, but before he can continue his mission, he’s pulled into Mojoworld for a multi-picture deal he simply can’t refuse. And when WOLVERINE and SHADOWCAT investigate the mysterious psionic disturbance, will they be able to escape the pull of Mojoworld before things SPIRAL out of control?",
    "downloadLinks": [
      {
        "Download Now": "https://getcomics.info/links.php//Voy3T8wNvbH9qvW3B0Rc90jWN867nhW3y3GquzWvkFQjqUUQtqJWR0fEMyrSMDPDK4sXt7GNU2i0+W6aRGlZk3qGIjQ9VwUGGnZSDImWA89xqse4yoKDs4WG4L2F8vwf9dG77GwCqC9V0XBqswCZE/3dsPyJAjSTSM38mfIhHc=:XKsZImoPVSVhhQVxDngX7g==",
      },
      {
        "UFILE": "https://getcomics.ufile.io/l1pd2tu9",
      },
      {
        "MEGA": "https://getcomics.info/links.php/lcLQvI5mz6lUmX6v17K9gVK7T1ttGUECeXUGuDMarmGo42oF6SiLTRtLFykhCD+SppVVjslDXJn1Rx0e5aKYDZO1CC9/uKnGp0dwoiDTnT8=:+vkz0rUgnn2L8zRnOPn/Lg==",
      },
      {
        "MEDIAFIRE": "https://getcomics.info/links.php/Fn6O/d7g6ZSJ31MBHvNZf/KuA2yQ4gi5ccA5JBv+O7xmPEXn8ZK/xeHwXMoPYfIyPEsLKWQrS0tlYAPTYF5QPPXPWk0CmwdfjhVp02hOnzhdDh/L2EgMWyI3YYKQ7Sq+JbYrG9ScD+K+J737RhxhQxARL2PfV0KetwqiQVU5UrDPppiSg2ljZZadYUFyGtLK:FgMNkeKlePyEjDbx6YkoEw==",
      },
      {
        "ZIPPYSHARE": "https://www3.zippyshare.com/v/eZ7m6ljt/file.html",
      },
      {
        "Read Online": "https://readcomicsonline.ru/comic/xmen-legends-2022/3",
      },
    ],
    "info": {
      "ImageFormat": "JPG",
      "Size": "37 MB",
      "Year": "2022",
    },
    "title": "X-Men – Legends #3 (2022)",
  },
  {
    "coverPage": "https://i0.wp.com/getcomics.info/share/uploads/2022/10/Wolverine-26-2022.jpg?fit=400%2C608&ssl=1",
    "description": "THE MARK OF THE BEAST! On defense or offense, WOLVERINE is Krakoa’s weapon. But that doesn’t mean he attacks where BEAST aims…and McCoy’s latest ask is going to set Logan on a startling and revelatory path set to redefine his future! A perfect jumping-on point for new readers…and the culmination of story threads from the start of this volume no continuing reader can afford to miss!",
    "downloadLinks": [
      {
        "Download Now": "https://getcomics.info/links.php/VB/EqoqPzfLwJ7H3jRg84VewA/ork04b6ErTQejsqI7ZQ6akPolB+rK8m3/vCOuu5U8P5OoXJgq3LcP9S2WqyvcRxC8i5y+i6jVSwWQPdxBzfvSItf9srdyagDWkoY5LppRosbk8wizNNfYNIsMgUIO3tjDnzhBaL/CgbuIo/cU=:TrIElPbw5lIO/VpvUQD0iw==",
      },
      {
        "UFILE": "https://getcomics.ufile.io/26qegku0",
      },
      {
        "MEGA": "https://getcomics.info/links.php/ndBEtKBNXMJmxzcmoMdT2uaLfd/xdZEgHPhqzCfofHPjWmot/iq2iiU/x/6eG7Y8aY6q0GXQpANFIau/ewXQyXgNM/xsYyzOw+vIxjjUVu0=:d1DKU35Z1i480LUNEaGlpQ==",
      },
      {
        "MEDIAFIRE": "https://getcomics.info/links.php/dhayU6P8bWN4B1gS07Rg3bHAsq2gy66SZkjMdK1KdzJx0V0GPMdWZ08UYsKvQ47XtwLcBF1baSXPqJDN38u36E9p2MPPg2PBVh51wCciJVksx0/gzRKGVbdjFmUzd7R0/0HBQ38JLuZYVUNs6/EoJyjc2XI2l1MuJSKLl6PnofQ=:7Z1z0UGuoV42NNoP8CfcOg==",
      },
      {
        "ZIPPYSHARE": "https://www91.zippyshare.com/v/la34pLLz/file.html",
      },
      {
        "Read Online": "https://readcomicsonline.ru/comic/wolverine-2020/26",
      },
    ],
    "info": {
      "ImageFormat": "JPG",
      "Size": "37 MB",
      "Year": "2022",
    },
    "title": "Wolverine #26 (2022)",
  },
  {
    "coverPage": "https://i0.wp.com/getcomics.info/share/uploads/2022/10/Wild-Cards-The-Drawing-of-Cards-4-2022.jpg?fit=400%2C608&ssl=1",
    "description": "IS HE ACE OR JOKER? A superhuman with an awesome gift…or a poor soul twisted by mutation? For Croyd Crenson, that question isn’t so easily answered. Sure, the Wild Card virus gave him powers…but they keep changing! Concluding the adaptation of the tragic first appearance of Roger Zelazny’s fan-favorite character—the Sleeper.",
    "downloadLinks": [
      {
        "Download Now": "https://getcomics.info/links.php/cf9hTPtK80B+kVR+b1S6uz7ojTJ1LYbwDdBL2OFgKMk0yzTx+PBax/PFqOU2uqk3GJUH28AmOSQ81J+zcRsy6M7BWMKy/xA3i6TGwHS4rvpXNfXrhypXblkPGmNc5lplfy7yYZWON/lhB8uodeWz1hNwQCuhchxW1sf2Qqf/gw2yNIyqVuN9ivZXNvmGKTuyX9SghTZnZ3hvxgJv43WVT0Ed07J9l54iiG2kn1D8UWI=:Sg4bhkH1/Bu9VuGTuIZoTw==",
      },
      {
        "UFILE": "https://getcomics.ufile.io/o68w1t20",
      },
      {
        "MEGA": "https://getcomics.info/links.php/ueXtFcvPeA5A63vNq57gHThmgW3rJeia81rt0vgSrTvyEqj2TIJlKK8qXor4u/QNZKHf3xpq9wgv0d5NVA3dmisj/q90rD/dGvcYnV79M6k=:2S4MbdYoHUCe3ZLEUNAU2Q==",
      },
      {
        "MEDIAFIRE": "https://getcomics.info/links.php/2/emYvd1GkZ390N4ipkDGJcbjd9eM5Z1VZ9HNSugw/PY+Hgg6z+WSrmGCuFobA3d7pNPsbouAC7JQjFBeg3mkkB88ZoBTcMFS2EMtxSaXuPXN/wqqmmPKZ3m9J9JrscF94c0l9ZqcBOocpVfQTEYQA6o2Y3lR3k6qEiGL8kbQCQ6Vozpi+fS4Y5D3yqCD0Qj/8MDbvV0G09gvNjky3SURQ==:erbJbq6RS2Z8fTlCOtTAlQ==",
      },
      {
        "ZIPPYSHARE": "https://www3.zippyshare.com/v/UXmX0K8N/file.html",
      },
      {
        "Read Online": "https://readcomicsonline.ru/comic/wild-cards-the-drawing-of-cards-2022/4",
      },
    ],
    "info": {
      "ImageFormat": "JPG",
      "Size": "42 MB",
      "Year": "2022",
    },
    "title": "Wild Cards – The Drawing of Cards #4 (2022)",
  },
  {
    "coverPage": "https://i0.wp.com/getcomics.info/share/uploads/2022/10/Venom-12-2022.jpg?fit=400%2C608&ssl=1",
    "description": "“VENOMWORLD” Continues! After the shocking events of VENOM #11, not only is Dylan Brock’s physical body in such grave condition that it may never recover, but his codex is as well. After a Hail Mary decision made by the Venom symbiote last issue, Dylan is about to see sides of symbiosis, the Klyntar and perhaps even his father that he has never even imagined!",
    "downloadLinks": [
      {
        "Download Now": "https://getcomics.info/links.php/JYPp+xRAQnTF2gsWZ/USdhjyhPVDtnM355V21XRNDcL9uRuZVng3Qw73ql5gbPQs4TVZgIHKnKxUjPmNDTjt0oVPyLuEPvBC/ic0nBCsGLktnW5YZDqDmTd+ReeIcDEJy/1biH9hZW3lQ2XJBhHWu5EOGLAMbm07ukvIXZ6baLs=:HBqE12MydRHp5PN6ClP0iA==",
      },
      {
        "UFILE": "https://getcomics.ufile.io/zxc2v9ca",
      },
      {
        "MEGA": "https://getcomics.info/links.php/oZEnSaofekJ0YY2uqXHDiz9SzZUe6xQqAIk+fZ1HpIVCrNBWcHBKORlBpZlzk1fcVOrNVMLZRV1Sj21gfLZnYncI5+HiP86gULtY8kxMCzI=:C7BD9qMBXlY8OmxX2Wqctw==",
      },
      {
        "MEDIAFIRE": "https://getcomics.info/links.php/VJuk6uVsp8wD6pCQ2MG2qqPW569AHz1mdec+GK04RCx7g2y9ZcvkhaeBJwEu72FUN9WaRNl8el8tu/hYY3Aq/UEc63qsqYcvchvhouPT7W2AdK5A3Bg3jNitWOerrAKr8ekE656kXvXGZj0AlKcI6s6rLM7bwVw0Eh/3NkZ32jI=:+OWioqvItE0P5uvJYoVdMQ==",
      },
      {
        "ZIPPYSHARE": "https://www91.zippyshare.com/v/1ov3gF0G/file.html",
      },
      {
        "Read Online": "https://readcomicsonline.ru/comic/venom-2021/12",
      },
    ],
    "info": {
      "ImageFormat": "JPG",
      "Size": "40 MB",
      "Year": "2022",
    },
    "title": "Venom #12 (2022)",
  },
  {
    "coverPage": "https://i0.wp.com/getcomics.info/share/uploads/2022/10/Ultraman-The-Mystery-of-Ultraseven-3-2022.jpg?fit=400%2C608&ssl=1",
    "description": "Past and present collide, with the future on the line—as ULTRAMAN TAKES ON ULTRASEVEN! It’s a knock-down, drag-out battle for the fate of the world—with a couple of giant Kaiju thrown in for good measure. There’s no room for anyone else in this fight…right?",
    "downloadLinks": [
      {
        "Download Now": "https://getcomics.info/links.php/rRgcnNskFSk6TPWn91JyUcPS6mAVjwlglx6dPQnjwL3Qcy1qXLuoukYU6+/to77cdca6jYPZp8WrQAOH2MGEunmwiz+G4VrKCWvqfJRUhz7s0JL0Emo1MBw0UvAHwWF3HTlF57BoBd/lX0MrlPR2N0QOCSNCmsAvKafS5lMPofQbz3GbxSPCKjvL8E93/nD2lYg5gJBtYs+ySwgHEJJmOw==:Rbz6NxUq+HQ6KDMEPdbGFQ==",
      },
      {
        "UFILE": "https://getcomics.ufile.io/y77anpot",
      },
      {
        "MEDIAFIRE": "https://getcomics.info/links.php/d1rCRFll4xOXDv6DEodHlInKRI2e42Qg9H/W/ELc8RC2n9Z73uTzGn3jFhP0Imuixw2l/VkfIO4GUOUOEyHAWr+Q+XtrSh/Qo+6tjQkXr+BlNxtfbjr1p0VHT4JxSxny/MY05G/qzzyQp/cr1G3vtNU95dbEJnjpSo4KvMA4NZC26eCTyMRYlZQBe4q0Mdkomz8PC/m/I8biRkIIwZn7dw==:p2qnCDvDYlDzTIk40bivTw==",
      },
      {
        "ZIPPYSHARE": "https://www91.zippyshare.com/v/U9nSxV4G/file.html",
      },
      {
        "Read Online": "https://readcomicsonline.ru/comic/ultraman-the-mystery-of-ultraseven-2022/3",
      },
    ],
    "info": {
      "ImageFormat": "JPG",
      "Size": "25 MB",
      "Year": "2022",
    },
    "title": "Ultraman – The Mystery of Ultraseven #3 (2022)",
  },
]
`;
